### ISSUE TITLE
iedorep review Sep 2022 - Errors

### ISSUE THREAD

**Issue by luisesanmartin** [worldbank/ietoolkit#289]
I tried running the most recent dev version of `iedorep` with real reproducibility packages. Found a few errors which I'm listing below:

- Do-files with comments that start with the characters `//` cause an error, as in
```
//	Set folder paths -----------------------------------------------------------
```
- This line in an input do-file also caused an error
```
	global do           		"$user_git/03 – Data/07_Covid19 HA Data/Replication Package/dofiles"
```
When I changed `global do` to `global do33` the error didn't appear anymore.

The error shown in both cases was this:
![image](https://user-images.githubusercontent.com/38444484/190278018-477370dc-dbf3-47eb-892a-310d65cd05e9.png)

- I also found this other error, but I don't know exactly what originates it
![image](https://user-images.githubusercontent.com/38444484/190278393-9d843ac6-6d87-447f-8988-bb971b7041c5.png)
The first time it showed, it appeared after I had run another do-file which left a dataset loaded in memory. After typing `clear` and trying again, it didn't show up. Then I tried generating a dummy dataset by typing `set obs 12` and `gen n = _n`, tried `iedorep master.do` again, and the same error showed. After typing `clear` again it didn't show up. I tried replicating it with another Stata session by typing again `set obs 12` and `gen n = _n`, and running `iedorep` on another input do-file, but it only seems to show when I try it with this particular master do-file that I can share with you

**Comment by bbdaniels** [comment]
> Do-files with comments that start with the characters // cause an error

I can't replicate this, something else must be happening here.

**Comment by bbdaniels** [comment]
Solved in c4262e573d8af627ed0151a9f9e707900b4acd7f. I was missing an instance of `ieboilstart` due to a whitespace typo (my fault) and then needed to add `estimates clear` to its replacement to avoid memory issues.
